searx/engines/bing.py

```diff
@@ -54,6 +54,7 @@ def request(query, params):
     params['url'] = base_url + search_path
     params['headers']['User-Agent'] = ('Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 '
                                        '(KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36')
+    params['headers']['Accept'] = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8'
```

Comment:
The same `Accept` header is already present in searxng: https://github.com/searxng/searxng/blob/7fc8d72889742399abb0d327e355ab350ddb395b/searx/engines/bing.py#L73
So, I'm quite confident this change does not break anything.
However, I did not test the fix myself locally.